<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouTubeDownloader</class>
 <widget class="QMainWindow" name="YouTubeDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Downloader</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>youtube.png</normaloff>youtube.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video Path:</string>
    </property>
    <property name="buddy">
     <cstring>pathEntry</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathEntry">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>620</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>10</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>YouTube video URL is placed here</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="statusTip">
     <string>enter video path here...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>625</y>
      <width>149</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Resolutions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Choose a resolution</string>
    </property>
    <property name="statusTip">
     <string>Available Resolutions</string>
    </property>
    <property name="title">
     <string>Resoluation</string>
    </property>
    <widget class="QRadioButton" name="res_144">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>144p</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="res_240">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>240p</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="res_360">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>360p</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="res_480">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>480p</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="res_720">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>720p</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>650</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="progress_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_size">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A MBs</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_size_label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>downloaded:</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_eta_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>540</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Remaining</string>
    </property>
   </widget>
   <widget class="QLabel" name="download_eta">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A seconds</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_to_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>save to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="destination_path">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>225</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>10</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>YouTube video URL is placed here</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="statusTip">
     <string>enter video path here...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse Destination Folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>pathEntry</tabstop>
  <tabstop>res_144</tabstop>
  <tabstop>res_240</tabstop>
  <tabstop>res_360</tabstop>
  <tabstop>res_480</tabstop>
  <tabstop>res_720</tabstop>
  <tabstop>download</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
